services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/CableReportService.kt

```diff
@@ -35,4 +35,6 @@ interface CableReportService {
     fun getApplicationInformation(applicationId: Int): AlluApplicationResponse
 
     fun cancel(applicationId: Int)
+
+    fun sendSystemComment(applicationId: Int, msg: String): Int
```

Comment:
Might not be in the scope of this pr but I'm often confused with the applicationId. Is it applicationId in haitaton or applicationId in Allu (alluId)? It seems that we are using applicationId to describe both. This should be addressed somehow, but maybe not now.

Reply:
The emergent convention seems to be that in the Allu client `applicationId` references the id in Allu and the Haitaton database id everywhere else. This kind of makes sense, since Allu API uses the term `applicationId` in some JSON objects to refer to their own id.

It would probably make more sense in Haitaton to rename it to `alluId` in the Allu client API.

Reply:
Allu has other IDs like IDs for comments and decision attachments, so `alluId` didn't seem like the correct name, at least for all occasions. I used `alluApplicationId`.